aladdin/commands/build_components/__init__.py

```diff
@@ -0,0 +1,8 @@
+from aladdin.commands.build_components import __main__
```

Comment:
If build_components is an aladdin command now... presumably, it won't be directly executed as a python program anymore.  Could __main__.py be renamed, since that has specific implications?  Or is there a reason that it shouldn't be?

Reply:
it can be renamed 👍